### ISSUE TITLE
SA1008 OpeningParenthesisMustBeSpacedCorrectly

### ISSUE THREAD

**Issue by mudnug** [DotNetAnalyzers/StyleCopAnalyzers#1093]
public static implicit operator string (BasicAuthorizationString item)
        {
            return item.Value;
        }

will trigger this rule, but implicit operators are formatted with this space by default.

**Comment by sharwell** [comment]
Hi @mudnug, welcome to the project. :smile: 

@pdelvo @vweijsters Do you know what StyleCop did in this case? Do you have a preference for the way we should handle it here?

I tested the default formatting, and it turns out that this behavior only occurs if the target type of the operator is a built-in type. For example, if you use `String` instead of `string`, Visual Studio's formatter will *remove* the space between `String` and `(`.

**Comment by sharwell** [comment]
My current vote is to leave this project as-is, and file a bug on the Roslyn project to make their default formatter consistent in this case.

**Comment by sharwell** [comment]
The underlying bug was fixed for the 1.1 release of Roslyn (not sure when that will be made public).

**Comment by dlemstra** [comment]
@sharwell I am experiencing the same issue with a `byte[]`. The code is formatted like this:
```C#
public static explicit operator byte[] (MagickImage image)
```
Instead of
```C#
public static explicit operator byte[](MagickImage image)
```

Is this a bug in Roslyn or should we fix the analyzer?
